src/pages/dashboard.js

```diff
@@ -0,0 +1,6 @@
+import { Heading } from '@chakra-ui/react';
+import React from 'react';
+
+const Dashboard = () => <Heading as="h2">Dashboard</Heading>;
```

Comment:
why are you using `as="h2"` if this is the title of the page?

Reply:
Fixed